File: app/src/main/java/com/ianarbuckle/dublinbushelper/models/stopinfo/StopInformation.java

package com.ianarbuckle.dublinbushelper.models.stopinfo;

import java.util.List;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class StopInformation {

    @SerializedName("errorcode")
    @Expose
    private String errorcode;
    @SerializedName("errormessage")
    @Expose
    private String errormessage;
    @SerializedName("numberofresults")
    @Expose
    private Integer numberofresults;
    @SerializedName("timestamp")
    @Expose
    private String timestamp;
    @SerializedName("results")
    @Expose
    private List<Result> results = null;

    public String getErrorcode() {
        return errorcode;
    }

    public void setErrorcode(String errorcode) {
        this.errorcode = errorcode;
    }

    public String getErrormessage() {
        return errormessage;
    }

    public void setErrormessage(String errormessage) {
        this.errormessage = errormessage;
    }

    public Integer getNumberofresults() {
        return numberofresults;
    }

    public void setNumberofresults(Integer numberofresults) {
        this.numberofresults = numberofresults;
    }

    public String getTimestamp() {
        return timestamp;
    }

    public void setTimestamp(String timestamp) {
        this.timestamp = timestamp;
    }

    public List<Result> getResults() {
        return results;
    }

    public void setResults(List<Result> results) {
        this.results = results;
    }

}

File: app/src/main/java/com/ianarbuckle/dublinbushelper/models/stopinfo/StopInformation.kt
package com.ianarbuckle.dublinbushelper.models.stopinfo

import com.squareup.moshi.Json

data class StopInformation(@Json(name = "errorcode") val errorcode: String? = "",
                           @Json(name = "errormessage") val errormessage: String? = "",
                           @Json(name = "numberofresults") val numberofresults: String? = "",
                           @Json(name = "timestamp") val timestamp: String? = "",
                           @Json(name = "results") val results: List<Result> = emptyList())
